src/frontend/src/pages/Cart.tsx

```diff
@@ -4,8 +4,6 @@ import { Frappe, SizeOptions, MenuItem } from '../types/Types';
 import { TestBases, TestExtras, TestMilks } from '../tests/TestServerData';
 import '../css/Cart.css'
 import { useNavigate } from 'react-router-dom';
-import { FrappeExtra } from '../types/Types';
```

Comment:
We don't want to change anything in Cart.tsx. Aka: this PR is trying to undo the changes I just made to this file